src/libs/topics-lib.js

```diff
@@ -1,8 +1,8 @@
 const _ = require("lodash");
 import { Kafka, ResourceTypes } from "kafkajs";
 
-export async function createTopics(brokerString, topicNamespace, topicsConfig) {
-  const topics = topicsConfig;
```

Comment:
Is the plural to singular change intentional?  If so, the function name should change.